tests/WebTest.php

```diff
@@ -192,6 +199,22 @@ public function testCookieHttpOnly()
         $this->assertEquals(true, $this->getCookie()->getHttpOnly());
     }
 
+    public function testSessionIDReuse()
+    {
+        $this->request("set.php?key=ok&value=yep");
+
+        $cookie = $this->getCookie();
+        $sessionID = $cookie->getValue();
+
+        $session = new SessionInstance(
+            $cookie->getName(),
+            null,
+            $sessionID
+        );
+
+        $this->assertEquals("yep", $session->getSet('ok'));
```

Comment:
I'm not clear on how this is testing the new session ID feature, can you explain?